### ISSUE TITLE
Java Mouse Click / Pen Click Detection Too Sensitive for Use with Touch or Pen-Based Displays

### ISSUE THREAD

**Issue by mesheets** [BrickBot/VisualNQC#1]
Java mouse click / pen click detection is too sensitive. While not as noticeable when using a mouse, it is rather problematic when using a pen, to the point that it is extremely difficult to get a click with the pen registered.

While this appears to be a problem with Java and not the application, the application needs some additional mouse event handling logic in order to reduce the sensitivity.
~ http://stackoverflow.com/a/6111571

In short, statments in the application such as "addMouseListener(new java.awt.event.MouseAdapter()...)" would likely need to be updated to something along the lines of the DragInsensitiveMouseClickListener() in the aforementioned Stack Overflow post.

([Originally posted to SourceForge](https://sourceforge.net/p/funsoftware/patches/2/))
